iOS/Objective C/LevelPlayDemo/DemoViewController.m

```diff
@@ -161,8 +154,8 @@ - (void) createBannerAd {
 
     // Create the banner view and set the ad unit id and ad size
```

Comment:
"Unlike Rewarded Videos there are no placements" under showInterstitialButtonTapped should be removed